cli/src/funTest/kotlin/OrtMainFunTest.kt

```diff
@@ -63,7 +64,18 @@ class OrtMainFunTest : StringSpec() {
 
     override suspend fun beforeSpec(spec: Spec) {
         configFile = createSpecTempFile(suffix = ".yml")
-        configFile.writeValue(OrtConfigurationWrapper(OrtConfiguration()))
+        configFile.writeValue(
+            OrtConfigurationWrapper(
+                OrtConfiguration(
+                    packageCurationProviders = listOf(
+                        PackageCurationProviderConfiguration(
+                            name = "File",
+                            config = mapOf("path" to projectDir.resolve("gradle/curations.yml").toString())
```

Comment:
`.path` is probably more speaking than `.toString()` as it matches the key name.